pkg/template/cloudconfig.go

```diff
@@ -116,6 +135,15 @@ func (c *CloudConfig) String() string {
 	return c.config
 }
 
-func defaultEtcdMultiCluster(baseDomain string) string {
-	return fmt.Sprintf("etcd1=etcd1.%s:2380,etcd2=etcd2.%s:2380,etcd3=etcd3.%s:2380", baseDomain, baseDomain, baseDomain)
+func nodeName(index int) string {
+	return fmt.Sprintf("etcd%d", index)
+}
+
+func defaultEtcdMultiCluster(baseDomain string, count int) string {
+	var cluster string
+	for i := 1; i < count+1; i++ {
```

Comment:
@calvix Should we start numbering `etcd` nodes from 0 or 1?

Reply:
from 1 is the right way